camp_fin/templates/camp_fin/contribution-detail.html

```diff
@@ -17,13 +17,44 @@ <h3>
             </h3>
         </div>
     </div>
+    {% if user.is_authenticated %}
+    <div class="row">
+        <div class="col-md-8">
+            <div class="alert alert-info" role="alert"> 
+                <h4>
+                    {{ transaction.full_name }} <small>{{ transaction.address }}, {{ transaction.city }}, {{ transaction.state }}</small>
+                </h4>
+
+                <form action="{{ request.path }}" method="post">
+                    <table id="redaction-form">
+                        <tr>
+                            <td>
+                                {{ form.redact }}
+                            </td>
+                            <td>
+                                <label for="id_redact">
+                                    {% if transaction.redact %}Uncheck this box to include the donor's name and address with contributions displayed across the site.{% else %}Check this box below to redact the donor's name and address from contributions displayed across the site. This will also prevent their contributions from appearing in site search results.{% endif %}
+                                </label>
+                            </td>
+                        </tr>
+                    </table>
+                    {% csrf_token %}
+                    <div align="center">
+                        <input type="submit" value="Submit" class="btn btn-primary" />
```

Comment:
Could we have this button say Redact or Unredact depending on state instead of using a checkbox?